src/cluster.c

```diff
@@ -1685,10 +1685,7 @@ void clusterRemoveNodeFromShard(clusterNode *node) {
     dictEntry *de = dictFind(server.cluster->shards, s);
     if (de != NULL) {
         list *l = dictGetVal(de);
-        listNode *ln = listSearchKey(l, node);
-        if (ln != NULL) {
-            listDelNode(l, ln);
-        }
+        listSearchDel(l, node);
```

Comment:
nit:

all of the other touch points don't have a space after the method params separator `,`.

Reply:
In all cases I was trying to keep the spacing from the listSearchKey for consistency. Historically they were both used.